README.md

```diff
@@ -3,8 +3,8 @@ cdn-definitions
 
 Data definitions for Red Hat's content delivery network.
 
-[![Build Status](https://travis-ci.org/release-engineering/cdn-definitions.svg?branch=master)](https://travis-ci.org/release-engineering/cdn-definitions)
-[![Coverage Status](https://coveralls.io/repos/github/release-engineering/cdn-definitions/badge.svg?branch=master)](https://coveralls.io/github/release-engineering/cdn-definitions?branch=master)
+![Build Status](https://github.com/jm-wk/cdn-definitions/actions/workflows/tests.yml/badge.svg)
+[![codecov](https://codecov.io/gh/jm-wk/cdn-definitions/branch/master/graph/badge.svg?token=PYTFAS3DTC)](https://codecov.io/gh/jm-wk/cdn-definitions)
```

Comment:
links to build status and codecov don't look ok - it's pointed to your username. But it should be pointed to `release-engineering` group.

Reply:
Thanks! I forgot to change these back to release-engineering.